sql/db-connection.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+function addElective($title, $description, $lecturer){
+
+	$host = "localhost";
+	$db = "db_name";
```

Comment:
Can you think of a name for the DB that describes the purpose and content of the database? `db_name` does not really give a lot of information about what we have inside and what we use it for currently.